content/open-access.md

```diff
@@ -0,0 +1,55 @@
+* value for money as publicly funded research is made accessible to the public
+* regaining public trust in science
+* a more informed and educated public
+* influence on policy which is increasingly important for climate change and medical advances
+
+![](../assets/figs/benefits-of-oa-new.jpg)
+
+These benefits have led to many funders now requiring outputs of the research they fund to be made available open access. For example, the [European Commission](http://ec.europa.eu/programmes/horizon2020/node/1031) requires all projects receiving Horizon 2020 funding to make sure that any peer-reviewed journal article they publish is open access to optimise the impact of publicly-funded scientific research.
+
+There are 2 main routes to open access publishing: Gold and Green. The Gold route refers to making research articles freely accessible immediately upon publication. This can be done by publishing through an open access journal or a hybrid journal which offers both open access and subscription content. This route often involves additional article processing charges (APCs), and are paid for by the University or research institute to which the researcher is affiliated, or by the funding agency supporting the research. This limits Gold Open Access publishing to institutions that can afford the APCs.
+
+The Green route refers to publishing in a traditional subscription journal as normal, but involves self-archiving of the peer-reviewed [Postprints](postprints.md) (and/or not peer-reviewed [Preprints](preprints.md)) by the authors in an institutional or external subject-based repository (try to avoid using a personal website which may lead to [reference rot](reference-rot.md)). No additional charges to the authors are incurred, but care needs to be taken on this route as journals and publishers may have [policies on self-archiving](sherpa.ac.uk/romeo) or require an [embargo](embargo.md) period before deposition in public repositories is allowed. This route may also still be restrictive in how people are allowed to share the research and redistribute it due to [Copyright](copyright.md), despite the fact that you can access the information in a postprint. The main problem is that [Copyright Transfer Agreements](copyright-transfer-agreements.md) often require the author to transfer *all* copyright. To promote permissive reuse of your research, read more about [Licensing](licensing.md).
+
+
+## Exercises
+
+* Consider what your definition of Open Access is. How does it compare to that of the Budapest Open Access Initiative?
+* Talk with your direct supervisors or colleagues about what level of Open Access you'd be comfortable committing to for future manuscripts and other research outputs.
+* Is there a preprint/postprint repository specific to your field? If there isn't a field-specific repository suitable to your research, is there a general Open Access server where you can comply with the Green route?
```

Comment:
Just wondering whether it fits here or is already part of preprints/postprints and people will click through to those. 

Probably not the latter, but there's no info here about repositories. Maybe drop this?